packages/models/src/eservice/eservice.ts

```diff
@@ -116,5 +138,6 @@ export const EService = z.object({
   riskAnalysis: z.array(RiskAnalysis),
   mode: EServiceMode,
   isSignalHubEnabled: z.boolean().optional(),
+  template: EServiceTemplateInfo.optional(),
```

Comment:
like I said in previous comment in the field  using only name `EServiceTemplate` is less ambiguous.